.gitignore

```diff
@@ -1,4 +1,5 @@
 bazel-*
+MODULE.bazel.lock
```

Comment:
Don't you typically want to commit the lockfile? The bazel [best practices](https://bazel.build/external/lockfile#best-practices) encourage it.